src/google/protobuf/json/json.h

```diff
@@ -69,6 +69,21 @@ inline absl::Status MessageToJsonString(const Message& message,
   return MessageToJsonString(message, output, PrintOptions());
 }
 
+// Converts from protobuf message to JSON and appends it to |json_output|. This is a
+// simple wrapper of BinaryToJsonString(). It will use the DescriptorPool of the
+// passed-in message to resolve Any types.
+//
+// Please note that non-OK statuses are not a stable output of this API and
+// subject to change without notice.
+PROTOBUF_EXPORT absl::Status MessageToJsonStream(const Message& message, 
+                                                 io::ZeroCopyOutputStream* json_output,
+                                                 const PrintOptions& options);
+
+absl::Status MessageToJsonStream(const Message& message, 
```

Comment:
This definition needs to be marked `inline` to prevent ODR violations. That should fix the errors that came up in the latest round of test runs.

Reply:
This definition has been be marked `inline`.

Reply:
@acozzette The ​unit test compilation failures seem unrelated to the changes in this PR.